decidim-participatory_processes/app/commands/decidim/participatory_processes/create_space_page.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+module Decidim
+  module ParticipatoryProcesses
+    class CreateSpacePage < Decidim::Command
+      # Public: Initializes the command.
+      #
+      # organization - A Decidim::Organization instance.
```

Comment:
I'd prefer to always use yardoc for consistency, specially in files that we're changing or adding

```suggestion
      # Initializes the command.
      #
      # @param organization [Decidim::Organization] a Decidim::Organization instance
```